### ISSUE TITLE
Counter Module

### ISSUE THREAD

**Issue by hishamco** [OrchardCoreContrib/OrchardCoreContrib.Modules#112]
Counter module counts how many visitors on your website. This module provides real time counting. All data saved to database.

https://www.drupal.org/project/counter

**Comment by Piedone** [comment]
I don't think nowadays something like this is too useful. You want to gather more info about users than simply counting page loads, and the various analytics services can do that. If you want to go full server-side (as opposed to e.g. Google Analytics requiring JS and the consent of users) then something like Azure Application Insights provides ample data.

**Comment by hishamco** [comment]
You're right, but as you said some needs some scripts or you have access to Cloud-based tools
